www/resources/views/snippets/optimizationchecklist.blade.php

```diff
@@ -1 +1,32 @@
-<!-- todo: a <table> goes here -->
\ No newline at end of file
+<table class="checklist">
+    <thead>
+        <tr>
+            <th>Request</th>
+            @foreach ($headers as $key => $text)
+            <th>
+                {{ $text }}
+                <br />
+                @if (isset($pageData[$key]) && $pageData[$key] >= 0 && $pageData[$key] <= 100)
+                    {{ $pageData[$key] }}%
+                @else
+                    N/A
+                @endif
+            </th>
+            @endforeach
+        </tr>
+    </thead>
+    <tbody>
+        @foreach ($rows as $row)
+        <tr class="table-cell-{{ $row['color'] ?? ($loop->even ? 'even' : 'odd')}}">
```

Comment:
Where does `$loop` come from?

Reply:
`$loop` is some Blade magic that helps working with loops
https://laravel.com/docs/9.x/blade#the-loop-variable